_topic_maps/_topic_map.yml

```diff
@@ -3110,6 +3110,8 @@ Topics:
     File: virt-installing-virtio-drivers-on-existing-windows-vm
   - Name: Installing VirtIO driver on a new Windows virtual machine
     File: virt-installing-virtio-drivers-on-new-windows-vm
+  - Name: vTPM encryption for Windows 11
```

Comment:
Please refrain from using the word "encryption" to define our current vTPM implementation.
A typical use of a TPM chip, whether physical or virtual, is indeed to store a disk encryption key.
However, we do not currently support vTPM state persistence, so using it to store any encryption key would result in that key being lost on the next VM shutdown.
What happens to a VM after a TPM device is added to it is just that, it will see a TPM device and be able to use it.

Also, while Windows 11 requires a TPM, a vTPM device can be added to any virtual machine running any operating system.
There is indeed limited benefit to having access to a volatile TPM device, and right now the only real life use-case might be installing Windows 11, but it's not limited to that.